config.js

```diff
@@ -9,6 +9,7 @@ var localConfig = extend(true, {
         // [Source, excluding bower_components]: [Destination].
         'document-register-element/build/document-register-element.max.js': config.LIB_DEST_PATH,
         'flipsnap/flipsnap.js': config.LIB_DEST_PATH,
+        'fxpay/dist/fxpay.min.js': config.LIB_DEST_PATH,
```

Comment:
If you have a non-min version, you can use it here so it's easier to debug. It'll get minified later.